docs/resources/team.md

```diff
@@ -4,3 +4,49 @@ description: Meet the SourceCred team!
 ---
 
 This page is under construction!
+
+
+# Dandelion
+## (they/them)
+
+### SourceCred Branch/Job Description: I'm the founder of SourceCred, and currently serve as its Temporary Benevolent Dictator (TBD). I also lead the dev and strategy branches of the project.
+
+### *I'm a genderqueer hacker, a joyful hacker, and a bad philosopher. I've long been fascinated by the intersection of economics, philosophy, and technology, and SourceCred is an attempt to synthesize those perspectives into a tool to build new, more equitable ways of working together. I also love snowboarding and partner dancing.*
+
+### Contact: decentralion, across platforms (GitHub, Discord, Twitter, etc)
+
+-photo-
+
+# LB 
+## (they/them)
+
+### SourceCred Branch/Job Description: Community Cultivation/Social Architect
+
+### *I'm a genderqueer artist and communicator. I got to be around for the conception of SourceCred, and later became a part of the team using my soft skills around interpersonal relationships and community structure. Now, I champion the Community Cultivation branch and work with our community to develop a wholehearted space that can scale with its values.*
+
+### Contact: LBS on Discord 
+
+-photo-
+
+# Seth
+## (he/him)
+
+### SourceCred Branch/Job Description: Creator of writings, hacky code, art, and thots on decentralized governance. Community builder, optimist, unlicensed therapist and cryptonaut.
+
+### *Here for the decentralized revolution, building a better alternative, and creating community with the amazing, talented people in SourceCred. When not working meditating, surfing, creating art and making conversation.*
+
+### Contact: s_ben on Discord, @zen_bacon (Twitter)
+
+-photo-
```

Comment:
![image of ben](https://github.com/sourcecred/docs/blob/master/images/team_photos/seth.png0